app/shared/landing-pages/SlantSection.js

```diff
@@ -0,0 +1,20 @@
+import { theme } from '../../../tailwind.config';
+
+export const DEFAULT_SLANT_SECTION_COLORS = ['rgba(0,0,0,0)', theme.extend.colors.primary.dark, 'rgba(0,0,0,0)'];
+export const SlantSection = ({
```

Comment:
nice. There is another variable here - the direction. They change that sometimes

Reply:
https://github.com/thegoodparty/gp-webapp/pull/158/commits/18fd0b6f205e9587d8d4b37f7847fdbcf29df93a